src/components/LearnerActivityTable/index.jsx

```diff
@@ -75,14 +74,14 @@ const InactivePastMonthLearnersTable = () => {
 
   return (
     <TableContainer
-      id="inactive-past-month-learners"
-      className="inactive-past-month-learners"
-      fetchMethod={EnterpriseDataApiService.fetchInactivePastMonthLearners}
-      columns={tableColumns}
+      id={props.id}
```

Comment:
nit: You could use es6 object deconstruction here by pulling out `id` and `activity` from the props above where we create the `tableColumns` variable:

```javascript
const { activity, id } = props;

const tableColumns = [...];
```